src/docs/guides/decoupled-cms/README.md

```diff
@@ -63,17 +66,13 @@ The newly created website should be able to run, and look like this:
 
 This will allow for the Razor Pages to be reloaded without the need to recompile them.
 
-- Modify the `<ItemGroup>` section like this:
+- Add a new `<ItemGroup>` section like this:
 
 ```xml
 <ItemGroup>
-  <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc1-10004" />
+  <PackageReference Include="OrchardCore.Application.Cms.Targets" Version="1.0.0-rc2-13450" />
```

Comment:
Can we made the version `1.0.*` to avoid change it until `1.1.*`

Reply:
That would move the actually used version though if the user uses the preview feed. I think that referencing a particular version in this guide is better even if we need to update it with new releases.

Reply:
@sebastienros you mean this should reference `OrchardCore.Application.Cms.Core.Targets` instead? Yeah, I see that.

Reply:
Yes this one, thanks

Reply:
Changed it.